lib/board.rb

```diff
@@ -13,4 +13,50 @@ def place_mark(mark, position)
     pos = position.to_i - 1
     @cells[pos] = mark
   end
+
+  def find_row(number)
+    # returns the entire "row" (1, 2, 3) top to bottom
```

Comment:
Why distinguish the rows?  Could instead have a rows method, which returns all rows?  Same below for columns, and diagonals.  Then you would be truly agnostic to the number of rows, columns, or diagonals.

Reply:
good point, I like this way a lot better. Fixed!